### PR TITLE
Emissions - Add test empty-list when no results

### PR DESCRIPTION
**Pull request by MartinSchmidt**
* Update code based on format rules
* Added test that ensures empty list when no results.

### CODE REVIEW

File: domains/emissions/Emissions.API/API/Services/EnergiDataService.cs [hunk 1]
```diff
@@ -38,31 +33,17 @@ public async Task<IEnumerable<MixRecord>> GetResidualMixPerHour(DateTime dateFro
...
-    string GetMixQueryAll(DateTime dateFrom, DateTime dateTo)
-    {
-
-        return $"dataset/DeclarationProduction?start={dateFrom:yyyy-MM-ddTHH:mm}&end={dateTo:yyyy-MM-ddTHH:mm}&columns=HourUTC,PriceArea,version,ProductionType,ShareTotal&timezone=UTC&filter=" + "{\"FuelAllocationMethod\":\"All\"}";
-    }
+    private static string GetMixQueryTotal(DateTime dateFrom, DateTime dateTo) => $"dataset/DeclarationProduction?start={dateFrom:yyyy-MM-ddTHH:mm}&end={dateTo:yyyy-MM-ddTHH:mm}&columns=HourUTC,PriceArea,version,ProductionType,ShareTotal&timezone=UTC&filter=" + "{\"FuelAllocationMethod\":\"Total\"}";
 
-    string GetEmissionsQuery(DateTime dateFrom, DateTime dateTo)
-    {
+    private static string GetMixQueryAll(DateTime dateFrom, DateTime dateTo) => $"dataset/DeclarationProduction?start={dateFrom:yyyy-MM-ddTHH:mm}&end={dateTo:yyyy-MM-ddTHH:mm}&columns=HourUTC,PriceArea,version,ProductionType,ShareTotal&timezone=UTC&filter=" + "{\"FuelAllocationMethod\":\"All\"}";
```
Comment (CodeReaper) [inline domains/emissions/Emissions.API/API/Services/EnergiDataService.cs:38]: Why are we using both string interpolation and string concatenation?
